backend/ideas/serializers/idea.py

```diff
@@ -40,6 +45,8 @@ class IdeaMinimalSerializer(serializers.ModelSerializer):
     organization = serializers.SerializerMethodField()
     url_slug = serializers.SerializerMethodField()
     hub_shared_in = serializers.SerializerMethodField()
+    number_of_comments = serializers.SerializerMethodField()
```

Comment:
Perhaps it is sensible to make a new serializer that inherits this one and add the 2 new fields to that one?